packages/@okta/vuepress-site/code/react/okta_react/index.md

```diff
@@ -346,14 +351,12 @@ export default withOktaAuth(class SignIn extends Component {
 The component for this route (LoginCallback) comes with `@okta/okta-react`. It handles token parsing, token storage, and redirecting to a protected page if one triggered the sign in.
 
 ### Connect the Routes
-Update `src/App.jsx` to include your project components and routes. `Security` is the component that controls the authentication flows, so it requires your OpenId Connect configuration. By default, `@okta/okta-react` redirects to Okta's sign in page when the user isn't authenticated.
+Update `src/App.js` to include your project components and routes. `Security` is the component that controls the authentication flows, so it requires your OpenId Connect configuration. By default, `@okta/okta-react` redirects to Okta's sign in page when the user isn't authenticated.
```

Comment:
I accept this change, but I feel switching from .js to .jsx is one of two notable mis-steps Facebook has made with React progress.  It's a pure removal of information.

`</soapbox>`

Reply:
The reason I changed it is because `.js` files are created by default. It's either "update App.js" or "delete App.js and create App.jsx" or "rename App.js to App.jsx". Using "update" was the easiest, so that's what I used. I'm not tied to it.

Reply:
No, since CRA gives us .js, this is the better option.  I just feel regret at the necessary acceptance.

Reply:
...and I wanted someone to hear my reluctance.  That was apparently important to me.